SampleDanger/ViewController.swift

```diff
@@ -649,6 +649,647 @@ class ViewController: UIViewController {
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         
+    // Do any additional setup after loading the view.
+  }
+  
+  
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: SampleDanger/ViewController.swift;
  Line: 1291;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)